WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -751,6 +751,11 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
         is SiteNavigationAction.OpenDashboardPersonalization -> activityNavigator.openDashboardPersonalization(
             requireActivity()
         )
+
+        is SiteNavigationAction.OpenBloganuaryNudgeOverlay -> {
+            // TODO thomashortadev: Implement Bloganuary nudge overlay
```

Comment:
Just to confirm: is this `TODO`/`AppLog` part of another PR or should we remove it?

Reply:
It is part of another PR, I'm keeping it since the work is being done in the feature branch.